packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -350,6 +350,11 @@ class FlutterPlugin implements Plugin<Project> {
             }
             variant.outputs[0].processResources.dependsOn(copyFlxTask)
         }
+        if (project.android.hasProperty("applicationVariants")) {
+            project.android.applicationVariants.all addFlutterDeps
+        } else {
```

Comment:
Do we want to guard this with an `if (project.android.hasProperty("libraryVariants"))` as well, or do we know that Android projects without applicationVariants always have libraryVariants?

Reply:
We want to fail fast, if we cannot add flutter dependencies. We do that now. Leaving it, as discussed off-line.